tests/analysis.spec.ts

```diff
@@ -88,14 +88,14 @@ describe('Functional test of analysis', () => {
         expect(new Set(bundle.analysisResults.coverage)).toEqual(
           new Set([
             {
-              files: 2,
```

Comment:
q: Why reorder those? Since this is a set, I would expect it to be order-insensitive?

Reply:
Good catch - I'll refactor these commits and remove this change.